config/jobs/kubernetes/test-infra/test-infra-presubmits.yaml

```diff
@@ -110,7 +110,7 @@ presubmits:
       preset-kind-volume-mounts: "true"
     spec:
       containers:
-      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20230509-e4070b06c0-test-infra
+      - image: gcr.io/k8s-staging-test-infra/krte:v20230421-ec4335b54b-master
```

Comment:
this image is only supported for testing kind itself (see the readme), and kubekins-e2e should work fine on containerd nodes, we're already using containerd nodes in the prow.k8s.io clusters for O(years) now?

this change broke the job https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/test-infra/29495/pull-test-infra-integration/1656409275724992512

the scripts are different

Reply:
Oops. Reverting.